codes/classical/bits/binary_linear.yml

```diff
@@ -20,13 +23,24 @@ protection: |
     G H^{\text{T}} = 0 \mod 2~.
   \end{align}
 
+  There is a homological construction for some classical linear codes \cite{arXiv:quant-ph/0605094}. However, not all linear codes have homological construction, there is a family of classical homological codes saturating the Hamming bound.
```

Comment:
Sorry about this... there is already an entry code_id: homological_classical that has this construction. I forgot to remove this assignment from the spreadsheet. Please integrate this into that file.